### PR TITLE
P drepass update user and organization

### PR DESCRIPTION
**Pull request by 2HEADSs**
Refactoring functionality to update users and organizations,
After the merge it will remove the admin router - I made a trello task for myself
Everything is tested.
I made a small change with quotes to be consistent like Hristo Yonkov's code

### CODE REVIEW

File: src/services/organizationService.js [hunk 1]
```diff
@@ -34,45 +34,42 @@ async function registerOrganization(organizationData) {
...
-    : existingOrganization.region;
-  existingOrganization.address = requestBody.address
-    ? requestBody.address
-    : existingOrganization.address;
+  if (!existingOrganization) {
+    throw new Error('Organization not found');
+  }
+
+  for (let key of Object.keys(requestBody)) {
+    if (key == 'email' && requestBody[key] == '') {
```
Comment (Getsov) [inline src/services/organizationService.js:58]: what if key == 'isDeleted' from hacked front end and !isAdmin ?

File: src/services/userService.js [hunk 2]
```diff
@@ -48,28 +48,27 @@ async function getById(id) {
...
-    ? requestBody.firstName
-    : existingUser.firstName;
-  existingUser.lastName = requestBody.lastName
-    ? requestBody.lastName
-    : existingUser.lastName;
-  existingUser.region = requestBody.region
-    ? requestBody.region
-    : existingUser.region;
+  for (let key of Object.keys(requestBody)) {
+    if (key == 'email' && requestBody[key] == '') {
```
Comment (Getsov) [inline src/services/userService.js:62]: Same here with isDeleted

Comment (2HEADSs) [reply src/services/userService.js:62]: isAdmin cames after parsing token
isDeleted will be added

File: src/services/organizationService.js (hunk 1 above)
Comment (2HEADSs) [reply src/services/organizationService.js:58]: isAdmin cames after parsing token
isDeleted will be added

**Review by Getsov — APPROVED** [review]

**Pull request merged**